src/Security/User/SwitchUserButtonGenerator.php

```diff
@@ -95,17 +88,12 @@ public function generateSwitchUserButton(array $row, string $href, string $label
         $tokenUser = $this->tokenStorage->getToken()->getUser();
 
         if ($tokenUser->getUsername() === $user->username) {
-            return '';
+            return Image::getHtml(preg_replace('/\.svg$/i', '_.svg', $icon));
         }
 
-        $url = $this->router->generate('contao_backend', [
-            '_switch_user' => $user->username,
-        ]);
+        $url = $this->router->generate('contao_backend', ['_switch_user' => $user->username]);
+        $title = StringUtil::specialchars($title);
 
-        return $this->twig->render('@ContaoCore/Backend/switch_user.html.twig', [
-            'url' => $url,
-            'title' => $title,
-            'image' => Image::getHtml($icon, $label),
-        ]);
+        return sprintf('<a href="%s" title="%s">%s</a>', $url, $title, Image::getHtml($icon, $label));
```

Comment:
I'm totally against to remove the template. Now we have again html somewhere in the logic.